packages/jest-runner/worker.ts

```diff
@@ -215,15 +232,91 @@
 		});
 	}
 
-	// noinspection JSUnusedLocalSymbols
 	private async runHooks(
 		hookType: string,
-		block: Circus.DescribeBlock | Circus.TestEntry,
+		block: Circus.DescribeBlock | Circus.TestEntry, // TODO: kill testEntry
 		ancestors: string[],
-		// eslint-disable-next-line @typescript-eslint/no-unused-vars
 		shouldRunInAncestors = false
 	) {
-		// TODO: Implement
+		let correctHooks: Circus.Hook[] = [];
+		if (block.type === "describeBlock") {
```

Comment:
Should always be `describeBlock`, shouldn't it?

Reply:
Yes, done!